apps/web/cypress/support/commands.ts

```diff
@@ -9,6 +9,10 @@ Cypress.Commands.add('getBySelectorLike', (selector, ...args) => {
   return cy.get(`[data-test*=${selector}]`, ...args);
 });
 
+Cypress.Commands.add('clickWorkflowNode', (selector: string, ...args) => {
+  return cy.getByTestId(selector).click({ force: true });
+});
```

Comment:
A small suggestion feel free to skip, it seems more like in terms of functionality, and also might be used for different usecases later: 
```suggestion
Cypress.Commands.add('forceClick', (selector: string, ...args) => {
  return cy.getByTestId(selector).click({ force: true });
});
```

Reply:
I do not like this one because... I think we should avoid force click as much as possible to make the tests as real as possible. Thats why I named this command very specific 😄